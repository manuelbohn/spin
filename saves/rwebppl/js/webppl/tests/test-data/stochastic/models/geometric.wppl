var geometric = function(p) {
  return flip(p) ? 1 + geometric(p) : 1;
};

var model = function() {
  return geometric(0.5);
};
