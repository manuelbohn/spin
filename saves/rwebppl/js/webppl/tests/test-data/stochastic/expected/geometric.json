{
  "hist": {
    "1": 0.5004887585532747,
    "2": 0.25024437927663734,
    "3": 0.1251221896383187,
    "4": 0.06256109481915935,
    "5": 0.03128054740957966,
    "6": 0.015640273704789837,
    "7": 0.007820136852394912,
    "8": 0.003910068426197456,
    "9": 0.001955034213098728,
    "10": 0.0009775171065493648
  },
  "MAP": {
    "val": 1,
    "score": -0.6921701409121187
  }
}
